doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -348,40 +352,38 @@ analysis on inductive or co-inductive objects (see :ref:`inductive-definitions`)
          induction y in x |-   *.
          Show 2.
 
-.. tacv:: induction @term with @bindings as @or_and_intropattern_loc using @term with @bindings in @goal_occurrences
-          einduction @term with @bindings as @or_and_intropattern_loc using @term with @bindings in @goal_occurrences
+.. tacv:: induction @term with @bindings @as_or_and_ipat using @term with @bindings in @goal_occurrences
+          einduction @term with @bindings @as_or_and_ipat using @term with @bindings in @goal_occurrences
 
    These are the most general forms of :tacn:`induction` and :tacn:`einduction`. It combines the
    effects of the with, as, using, and in clauses.
 
-.. tacv:: elim @term
-   :name: elim
+.. tacn:: elim @one_term_with_bindings {? using @one_term {? with @bindings } }
 
    This is a more basic induction tactic. Again, the type of the argument
-   :n:`@term` must be an inductive type. Then, according to the type of the
+   :n:`@one_term` must be an inductive type. Then, according to the type of the
```

Comment:
More basic / most similar to `induction`. The main difference between `elim` and `induction` and `case` and `destruct` is that `elim` and `case` do not introduce the variables in the goals they generate. There are other subtle differences but I don't know about them.

Reply:
Another difference is that `destruct` and `induction` generalize over hypotheses, whereas `case` (and I presume `elim` only look for occurrences in the goal.  Notably this means that if the term being case analyzed is a context variable, it's not cleared by `case`.

Reply:
`fix` and `cofix` tactics are meant for use with `case` and/or `elim`.  They are a) not needed for `inductive` or `destruct` and b) not otherwise useful?

Reply:
I don't understand your question (if you are talking about the `fix` and `cofix` tactics or term constructs). In any case, `fix` is low level. You don't need to call it directly if you use `elim` or `induction` (which insert an induction principle which itself uses `fix` internally). `cofix` is different because there is not yet any high-level tactic which would allow doing without (in the stdlib, but people actually working with co-inductive types generally use external helper libraries).